### DISCUSSION TITLE
How to use autodoc2_hidden_regexes ?

### DISCUSSION THREAD

**Discussion by FabriceSalvaire** [sphinx-extensions2/sphinx-autodoc2#57] [General]
Can we hide module and class members using `autodoc2_hidden_regexes` ?

**Comment by FabriceSalvaire** [comment]
Got it !
We must do
```
autodoc2_hidden_regexes = [
    r'.*\._module_logger',
    r'.*\._logger',
]
```
Would be more efficient to match against `short_name`.

**Comment by Gabriel-p** [comment]
@FabriceSalvaire would you happen to know how to use `autodoc2_hidden_regexes` to hide all attributes?
